Sources/PaymentButtons/PaymentButton.swift

```diff
@@ -331,4 +332,9 @@ public class PaymentButton: UIButton {
         image.draw(in: CGRect(x: 0.0, y: 0.0, width: size.width, height: size.height))
         return UIGraphicsGetImageFromCurrentImageContext()
     }
+
+    private func customizeAppearance() {
+        containerView.layer.borderColor = color == .white ? UIColor.black.cgColor : UIColor.clear.cgColor
+        containerView.layer.borderWidth = color == .white ? 2 : 0
```

Comment:
What does Android use for border width? Looks like it's 1px but I'm not sure because the Merchant's image of the buttons you shared in Slack is very small. Figma designs show 1px https://www.figma.com/design/gn4AXdmJUOMVlrCMpyhfrf/QL---Checkout-Buttons?node-id=1-73&t=C9Kq3Q5LMzl7DqsD-0

Reply:
Ah thank you! I saw 2 in Android. But it did look a bit thick to me on screenshot for iOS.